web-app/src/games/games.controller.ts

```diff
@@ -26,16 +26,21 @@ export class GamesController {
   @Post('')
   @Redirect('/')
   async createNewGame(
-    @Body() gameParams: { size?: number; fromFile?: boolean },
+    @Body()
+    gameParams: { size?: number; fromFile?: boolean; soloMode?: string },
     @Req() req: Request,
   ) {
     let newGame: Game;
     if (gameParams.fromFile) {
-      newGame = await this.gameService.createNewGameFromFile(req.sessionID);
+      newGame = await this.gameService.createNewGameFromFile(
+        req.sessionID,
+        gameParams.soloMode === 'on',
```

Comment:
Why not use a boolean instead? Is it to allow to pass ia level later ?

Reply:
I have my answer from https://github.com/marmelab/Hex/pull/93#discussion_r823513758 and i let you choose if you want to switch to a bool val or keep the default val.